data/xml/Y18.xml

```diff
@@ -744,10 +746,10 @@
   <volume id="3">
     <meta>
       <booktitle>Proceedings of the 32nd Pacific Asia Conference on Language, Information and Computation: 5th Workshop on Asian Translation: 5th Workshop on Asian Translation</booktitle>
-      <editor><first>Chor Shing David</first><last>Li</last></editor>
-      <editor><first>Qin</first><last>Lu</last></editor>
-      <editor><first>Hans J.</first><last>Laadegard</last></editor>
-      <publisher>Association for Computational Linguistics</publisher>
+      <editor><first>Stephen</first><last>Politzer-Ahles</last></editor>
```

Comment:
Why did you remove the `<publisher>` line only on the third volume? Should it be retained here, or removed on the other two volumes as well?

Reply:
Oops, that's my mistake from the copy-pasting. It should be retained here. Sorry about that!

Reply:
I can't edit the PR since it is a non-local branch. If you make the change and push it up to your branch, it will be reflected here.

Reply:
Thanks, I made the change on the file over on my branch just now and committed it, does that fix it now or is there something else I should do as well?